packages/twenty-front/src/modules/object-record/record-show/components/RecordShowContainer.tsx

```diff
@@ -129,114 +132,118 @@ export const RecordShowContainer = ({
-                          FieldMetadataType.Text,
-                        iconName: '',
-                        fieldMetadataId:
-                          labelIdentifierFieldMetadataItem?.id ?? '',
-                        label: labelIdentifierFieldMetadataItem?.label || '',
-                        metadata: {
-                          fieldName:
-                            labelIdentifierFieldMetadataItem?.name || '',
-                          objectMetadataNameSingular: objectNameSingular,
-                        },
-                        defaultValue:
-                          labelIdentifierFieldMetadataItem?.defaultValue,
-                      },
-                      useUpdateRecord: useUpdateOneObjectRecordMutation,
-                      hotkeyScope: InlineCellHotkeyScope.InlineCell,
-                    }}
-                  >
-                    <RecordInlineCell readonly={isReadOnly} />
-                  </FieldContext.Provider>
-                }
-                avatarType={recordIdentifier?.avatarType ?? 'rounded'}
-                onUploadPicture={
-                  objectNameSingular === 'person' ? onUploadPicture : undefined
-                }
-              />
-              <PropertyBox>
-                {isPrefetchLoading ? (
-                  <PropertyBoxSkeletonLoader />
-                ) : (
-                  inlineFieldMetadataItems.map((fieldMetadataItem, index) => (
-                    <FieldContext.Provider
-                      key={objectRecordId + fieldMetadataItem.id}
-                      value={{
-                        entityId: objectRecordId,
-                        maxWidth: 200,
-                        recoilScopeId: objectRecordId + fieldMetadataItem.id,
-                        isLabelIdentifier: false,
-                        fieldDefinition:
-                          formatFieldMetadataItemAsColumnDefinition({
-                            field: fieldMetadataItem,
-                            position: index,
-                            objectMetadataItem,
-                            showLabel: true,
-                            labelWidth: 90,
-                          }),
-                        useUpdateRecord: useUpdateOneObjectRecordMutation,
-                        hotkeyScope: InlineCellHotkeyScope.InlineCell,
-                      }}
-                    >
-                      <RecordInlineCell
-                        loading={loading || recordLoading}
-                        readonly={isReadOnly}
-                      />
-                    </FieldContext.Provider>
-                  ))
-                )}
-              </PropertyBox>
-              <RecordDetailDuplicatesSection
-                objectRecordId={objectRecordId}
-                objectNameSingular={objectNameSingular}
-              />
-              {relationFieldMetadataItems?.map((fieldMetadataItem, index) => (
+  const summary = (
```

Comment:
Code smell, using a function is a sign that the component should be broken down in multiple smaller components